nomt/src/lib.rs

```diff
@@ -335,6 +370,38 @@ impl Nomt {
         Ok((new_root, commit.witness, commit.witnessed_operations))
     }
 
+    /// Perform a rollback of the last `n` commits.
+    ///
+    /// This function assumes no other sessions are active.
```

Comment:
The documentation is somewhat unclear here. In particular the meaning of 'no other sessions' is unclear, since this function is on `Nomt`, not `Session`.

Should it say "This function assumes no sessions are active"?